protos/event.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
```

Comment:
This looks all non-controversial. 
One question is how to structure protos. All in one file for core?

What is a little tricky - getting include path right on the c++ side.
I suggest to merge https://github.com/farm-ng/farm-ng-core/pull/8 first and then place this on top of it.